modules/combined/test/tests/frictional_contact/single_point_2d/single_point_2d.i

```diff
@@ -41,17 +35,16 @@
   [../]
 []
 
-[SolidMechanics]
-  [./solid]
-    disp_x = disp_x
-    disp_y = disp_y
-    save_in_disp_x = saved_x
-    save_in_disp_y = saved_y
-    diag_save_in_disp_x = diag_saved_x
-    diag_save_in_disp_y = diag_saved_y
+[Modules/TensorMechanics/Master]
+  [./all]
+    add_variables = true
+    strain = FINITE
+    save_in = 'saved_x saved_y'
```

Comment:
@bwspenc may know more but I'm not sure why these even have the save_in/ diag_save_in in the first place.  There is no problem block defining reference residual.  These should be able to removed from here and from the AuxVariables. Moreover, the use of save_in is deprecated and should not be used.

Reply:
We don't need diag_save_in anymore.  We do need the save_in variables because we're using them to compute reaction forces in several postprocessors. @recuero and I discussed this, and he'll switch to tagging in a follow-on PR.